### PR TITLE
Fix some django anti-patterns

### PR DESCRIPTION
**Pull request by code-review-doctor**
I found this problem during auditing your codebase. [Read the full report](https://django.doctor/llazzaro/django-scheduler)

### CODE REVIEW

File: schedule/models/events.py [hunk 1]
```diff
@@ -637,13 +637,12 @@ def hours(self):
     def get_absolute_url(self):
         if self.pk is not None:
             return reverse(
-                "occurrence",
-                kwargs={"occurrence_id": self.pk, "event_id": self.event.id},
+                "occurrence", kwargs={"occurrence_id": self.pk, "event_id": self.event_id}
```
Comment (django-doctor) [inline schedule/models/events.py:640]: Django caches foreignkey ids, so doing this means one less db read. [read more](https://django.doctor/advice/C9001)

File: schedule/views.py [hunk 2]
```diff
@@ -393,7 +393,7 @@ def convert(ddatetime):
     # event, using the "event_id" or the occurrence with the specified "id".
     # for more info https://github.com/llazzaro/django-scheduler/pull/169
     i = 1
-    if Occurrence.objects.all().count() > 0:
+    if Occurrence.objects.all().exists():
```
Comment (django-doctor) [inline schedule/views.py:396]: using `if queryset.exists()` is more efficient than checking `if queryset.count() > 0` [read more](https://django.doctor/advice/C9002)

**Comment by llazzaro** [comment]
@django-doctor the build failed, could you check it?

bot or not?

**Comment by django-doctor** [comment]
50% bot. 50% human :)

I used https://django.doctor to created the PR, using the "auto fix" tab

image

Is the failure related to my change? I'm seeing

> File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
> File "/home/travis/build/llazzaro/django-scheduler/schedule/urls.py", line 1, in <module>
> from django.conf.urls import url
>ImportError: cannot import name 'url'

**Comment by django-doctor** [comment]
@llazzaro as far as I can tell, the failure on CI is related to files out of scope of this change e.g, schedule/models/events.py

**Pull request merged**